src/box/box.h

```diff
@@ -620,6 +649,23 @@ box_iproto_send(uint64_t sid,
 		const char *header, const char *header_end,
 		const char *body, const char *body_end);
 
+/**
+ * Sets an IPROTO request handler with the provided context for the
+ * given request type.
+ *
+ * \param request_type request type code from iproto_type enumeration
+ *                     (src/box/iproto_constants.h);
+ *                     use IPROTO_UNKNOWN for overriding the handler of unknown
+ *                     request types;
+ * \param handler IPROTO request handler; passing NULL resets the corresponding
+ *                handler
```

Comment:
It isn't mentioned in the design document that `NULL` can be used to uninstall the currently installed handler. Since the doc bot request only contains a link to the design document, you should update it.

Reply:
Updated the design document and comments.